### DISCUSSION TITLE
[Detector Support]: Coyetes as a object?

### DISCUSSION THREAD

**Discussion by templeowls21** [blakeblackshear/frigate#15345] [Detector Support] [answered]
### Describe the problem you are having

Bit of an odd question...sorry if wrong category. We have a problem with coyotes in my area so I want to be able to detect them.

There's not a specific "coyote" object type nor just an "animal" type. Do you think "dog" would do the trick to capture these?

Thanks!

### Version

n/a

### Frigate config file

```yaml
n/a
```

### docker-compose file or Docker CLI command

```yaml
n/a
```

### Relevant Frigate log output

```shell
n/a
```

### Install method

Docker Compose

### Object Detector

Coral

### Screenshots of the Frigate UI's System metrics pages

n/a

**Comment by hawkeye217** [comment, marked as answer]
You could certainly try `dog`. You may have to lower the min_score and threshold.

The default free model that Frigate ships with is based on the COCO dataset, so your mileage may vary with how well it performs.

**Reply by templeowls21** [reply]
Gotcha, Thanks!

**Reply by hawkeye217** [reply]
Happy to help.
